onnxruntime/python/tools/transformers/fusion_layernorm.py

```diff
@@ -121,24 +121,28 @@ def __init__(self, model: OnnxModel):
 
     def fuse(self, node, input_name_to_nodes: Dict, output_name_to_node: Dict):
         """
-        Layer Norm from Keras in Tensorflow:
-         +----------------------+
-         |                      |
-         |                      v                               (B)                             (B)             (A)
-        Add --> ReduceMean -->  Sub  --> Mul --> ReduceMean --> Add --> Sqrt --> Reciprocol --> Mul --> Mul --> Sub --> Add
-         |          |                                                                            |       ^              ^
-         |          |                                                                            |       |              |
-         |          +----------------------------------------------------------------------------|-------+              |
-         |                                                                                       v                      |
-         +-------------------------------------------------------------------------------------> Mul--------------------+
+        Layer Norm from Tensorflow model(using keras2onnx or tf2onnx):
+         +------------------------------------+
+         |                                    |
+         |                                    |
+       (Cast_1)                               |
```

Comment:
Cast will be removed by CastElimination (optimizer will use ORT level 1 optimization first). I think there is no need to change this function.

Reply:
Ah I see

Reply:
actually I found that ORT level 1 optimization does not remove those cast node